survey/3-work.yml

```diff
@@ -30,7 +30,7 @@ questions:
       - Slightly satisfied
       - Very satisfied
 
-  - label: Do you work on a side project?
+  - label: Do you work on a side projects?
```

Comment:
did you mean "Do you work on side projects"?

Reply:
yeah correct

Reply:
```suggestion
  - label: Do you work on side projects?
```